### PR TITLE
Let the data-blizzard begin.

### PR DESCRIPTION
**Pull request by prescod**
Fix one usability problem, create a task name and include as a requirement.

### CODE REVIEW

**Comment by davisagli** [comment]
@prescod looks like the tests are failing because the snowfakery distribution doesn't include the yml files the test is looking for.

**Comment by prescod** [comment]
That was way harder than it should have been! Essentially this commit consists of 3 lines of YAML, one line of requirements.txt and tons and tons and tons of test fixes, especially for Windows.

File: cumulusci/cumulusci.yml [hunk 1]
```diff
@@ -140,6 +140,9 @@ tasks:
         class_path: cumulusci.tasks.datadictionary.GenerateDataDictionary
         options:
             release_prefix: $project_config.project__git__prefix_release
+    generate_and_load_from_yaml:
+        class_path: cumulusci.tasks.bulkdata.generate_and_load_data_from_yaml.GenerateAndLoadDataFromYaml
+        group: Data Operations
```
Comment (davisagli) [inline cumulusci/cumulusci.yml:145]: Looks like the task docs need to be regenerated to include the new task

File: cumulusci/tasks/bulkdata/tests/test_generate_from_snowfakery_task.py [hunk 2]
```diff
@@ -14,17 +15,29 @@
     from cumulusci.tasks.bulkdata.generate_and_load_data_from_yaml import (
         GenerateAndLoadDataFromYaml,
     )
+    from snowfakery import data_generator_runtime
```
Comment (davisagli) [inline cumulusci/tasks/bulkdata/tests/test_generate_from_snowfakery_task.py:18]: These no longer need to be imported conditionally.

File: cumulusci/tasks/bulkdata/tests/test_generate_from_snowfakery_task.py [hunk 3]
```diff
@@ -237,19 +242,21 @@ def test_with_nonexistent_continuation_file(self):
         assert "does not exist" in str(e.exception)
 
     def test_generate_continuation_file(self):
-        with NamedTemporaryFile() as temp_continuation_file:
-            with NamedTemporaryFile() as temp_db:
-                database_url = f"sqlite:///{temp_db.name}"
+        from snowfakery import version
+
+        print(version.version)
```
Comment (davisagli) [inline cumulusci/tasks/bulkdata/tests/test_generate_from_snowfakery_task.py:247]: Remove

File: cumulusci/tasks/bulkdata/tests/test_generate_from_snowfakery_task.py [hunk 4]
```diff
@@ -42,8 +55,7 @@ def test_no_options(self):
             _make_task(GenerateDataFromYaml, {})
 
     def test_simple(self):
-        with NamedTemporaryFile() as t:
-            database_url = f"sqlite:///{t.name}"
+        with temp_sqlite_database_url() as database_url:
```
Comment (davisagli) [inline cumulusci/tasks/bulkdata/tests/test_generate_from_snowfakery_task.py:58]: Personally I probably would have set up database_url as a pytest fixture (though that does require switching away from unittest.TestCase)

File: cumulusci/cumulusci.yml (hunk 1 above)
Comment (prescod) [reply cumulusci/cumulusci.yml:145]: https://github.com/SFDO-Tooling/CumulusCI/pull/1659/commits/819742de10afde65cafa3401b2dc979a4f71a3af

File: cumulusci/tasks/bulkdata/tests/test_generate_from_snowfakery_task.py (hunk 2 above)
Comment (prescod) [reply cumulusci/tasks/bulkdata/tests/test_generate_from_snowfakery_task.py]: https://github.com/SFDO-Tooling/CumulusCI/pull/1659/commits/c9f7e87ed50d4bcf2e2797c54b8dd01ab1e41dbc

File: cumulusci/tasks/bulkdata/tests/test_generate_from_snowfakery_task.py [hunk 5]
```diff
@@ -1,33 +1,42 @@
...
-    )
-except ImportError:
-    snowfakery = None
+import snowfakery
+from cumulusci.tasks.bulkdata.generate_and_load_data_from_yaml import (
+    GenerateAndLoadDataFromYaml,
+)
+from snowfakery import data_generator_runtime
 
 if snowfakery:
```
Comment (davisagli) [inline cumulusci/tasks/bulkdata/tests/test_generate_from_snowfakery_task.py:19]: This condition can go too

File: cumulusci/tasks/bulkdata/tests/test_generate_from_snowfakery_task.py (hunk 3 above)
Comment (prescod) [reply cumulusci/tasks/bulkdata/tests/test_generate_from_snowfakery_task.py]: https://github.com/SFDO-Tooling/CumulusCI/commit/df3e20d0e96823bf6055f61ee4c14056c3d12385

File: cumulusci/tasks/bulkdata/tests/test_generate_from_snowfakery_task.py (hunk 5 above)
Comment (prescod) [reply cumulusci/tasks/bulkdata/tests/test_generate_from_snowfakery_task.py]: https://github.com/SFDO-Tooling/CumulusCI/pull/1659/commits/2814208b6099beb23b5812d6d36207792fdff931

**Pull request merged**